server/container/guice/guice-common/pom.xml

```diff
@@ -152,6 +152,12 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>metrics-logger</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
+            <version>1.18.0</version>
```

Comment:
Don't you to declare new dependency inside `james-project/pom.xml`? I saw many dependencies declared inside`james-project/pom.xml` and then being used in different projects without specifying version

Reply:
we don't do when the depency is used only once. When it will be reused, we will move the version on the parent project.